### PR TITLE
Compatibility: INSERT INTO ... SELECT

### PR DESCRIPTION
**Pull request by haying**
Removed temp table for storing returned id, use plpgsql local variable instead

### CODE REVIEW

File: src/ports/postgres/modules/convex/test/lmf.sql_in [hunk 1]
```diff
@@ -100016,28 +100016,34 @@ COPY mlens100k (user_id, movie_id, rating) FROM STDIN;
 12	203	3
 \.
 
-DROP TABLE IF EXISTS _temp_model_id CASCADE;
-CREATE TEMPORARY TABLE _temp_model_id (id INTEGER);
+CREATE FUNCTION check_rmse()
```
Comment (haradh1) [inline src/ports/postgres/modules/convex/test/lmf.sql_in]: Can you put a comment to explain why we need a function?

Comment (haying) [inline src/ports/postgres/modules/convex/test/lmf.sql_in]: Forgot to check it in. Fixed

**Comment by cwelton** [comment]
This raises some longer term questions for me around model management.  

Linear regression:  We return the results directly, can be used as an aggregate.
Logistic regression: We return the results directly, but cannot be used as an aggregate.
K-means: Results written out to a user specified output table
Association Rules: Results written to system created tables in a user specified output schema
Decision Trees: Results written to user specified table with system generated auxiliary tables and tracked in a global model table by the user specified id.
SVD: Results written to a user specified output table returning a system generated id that references the model.

Which is to say... we have no consistency in terms of expected API.

**Comment by haying** [comment]
I noticed this. Apparently we do not have enough guidelines for developers currently. My feeling is that some developer guidelines on this is needed, although some constraints like most main modules have to call a fixed set of API or SQL macros to produce user output could be better.

**Comment by cwelton** [comment]
Agreed.  That was what I was trying to kick off early this release process, but it fell a little flat.  We'll have to take another stab at it in the next release.

**Pull request closed without merging**